README.md

```diff
@@ -1102,6 +1103,33 @@ frame.at_css("//a[text() = 'Log in']") # => Node
 #### select
 
 
+## Tracing
+
+You can use `tracing.start` and `tracing.stop` to create a trace file which can be opened in Chrome DevTools or [timeline viewer](https://chromedevtools.github.io/timeline-viewer/).
+
+```ruby
+browser.tracing.start(path: "trace.json")
+browser.go_to("https://www.google.com");
```

Comment:
Remove `;` please, if tracing is implemented in the page it's better to use page here rather than browser. I'm gonna rewrite README soon to make it clear.